config/jobs/image-pushing/k8s-csi-proxy.yaml

```diff
@@ -0,0 +1,30 @@
+postsubmits:
```

Comment:
@pohly has a PR to autogenerate this for all the csi repos in https://github.com/kubernetes/test-infra/pull/17741.

I've asked him to cover csi-proxy in his PR.

Reply:
got it. Thanks!